### PR TITLE
apiserver/README: update to 2018 for compatibility

### PR DESCRIPTION
**Pull request by nikhita**
/shrug
/assign sttts 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by neolit123** [comment]
there is a double space after `guarantee.`  :memo: :upside_down_face:

**Comment by nikhita** [comment]
> there is a double space after guarantee

@neolit123 Thanks for catching that, fixed. :)

File: staging/src/k8s.io/apiserver/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ This library contains code to create Kubernetes aggregation server complete with
 
 There are *NO compatibility guarantees* for this repository, yet.  It is in direct support of Kubernetes, so branches
 will track Kubernetes and be compatible with that repo.  As we more cleanly separate the layers, we will review the
-compatibility guarantee.  We have a goal to make this easier to use in 2017.
+compatibility guarantee. We have a goal to make this easier to use in 2018.
```
Comment (sttts) [inline staging/src/k8s.io/apiserver/README.md]: in the future :)

Comment (nikhita) [reply staging/src/k8s.io/apiserver/README.md]: haha, fixed. :)

**Comment by neolit123** [comment]
/lgtm
:tada:

**Pull request merged**